apollo-backend/src/main/java/io/logz/apollo/blockers/BlockerService.java

```diff
@@ -29,13 +30,15 @@
 
     private final BlockerDefinitionDao blockerDefinitionDao;
     private final BlockerInjectableCommons blockerInjectableCommons;
+    private final StackService stackService;
     private final Map<String, Class<? extends BlockerFunction>> blockerTypeNameBindings;
     private final Reflections reflections;
 
     @Inject
-    public BlockerService(BlockerDefinitionDao blockerDefinitionDao, BlockerInjectableCommons blockerInjectableCommons) {
+    public BlockerService(BlockerDefinitionDao blockerDefinitionDao, BlockerInjectableCommons blockerInjectableCommons, StackService stackService) {
```

Comment:
Please move StackService to BlockerInjectableCommons. To make things simpler - nothing else should be injected to BlockerService, ever.